### PR TITLE
Inject the EntityManager instead of the factory in DoctrineExtractor

### PR DESCRIPTION
**Pull request by dunglas**
Follows symfony/symfony#27829

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "symfony/dependency-injection": "~2.7|~3.0|~4.0",
         "doctrine/dbal": "^2.5.12",
         "jdorn/sql-formatter": "^1.2.16",
-        "symfony/doctrine-bridge": "~2.7|~3.0|~4.0",
+        "symfony/doctrine-bridge": "^4.2",
```
Comment (dunglas) [inline composer.json]: We'll be able to use feature detection before merging.

Comment (stof) [reply composer.json]: aren't we already able to do it ?

Comment (dunglas) [reply composer.json]: Not sure if there is a new class yet.

**Comment by kimhemsoe** [comment]
Dont know the details, but i am guessing this should be merged?

File: composer.json (hunk 1 above)
Comment (dunglas) [reply composer.json]: done

**Comment by dunglas** [comment]
Ready for a last review. It should be merged before the release of 4.2, because without this patch DoctrineBundle triggers a deprecation.

**Comment by kimhemsoe** [comment]
@dunglas Thanks

**Pull request merged**